apps/react-18-tests-v8/webpack.config.js

```diff
@@ -12,22 +12,53 @@ module.exports = () => {
     },
     resolve: {
       extensions: ['.tsx', '.ts', '.js'],
-      plugins: [
-        new TsconfigPathsPlugin({
-          configFile: path.resolve(__dirname, '../../tsconfig.base.json'),
-        }),
-      ],
       alias: {
         react: path.resolve(__dirname, './node_modules/react'),
         'react-dom': path.resolve(__dirname, './node_modules/react-dom'),
+        ...getResolveAlias(),
       },
     },
     module: {
       rules: [
         {
           test: /\.tsx?$/,
           exclude: /node_modules/,
-          use: 'swc-loader',
+          use: {
+            loader: 'esbuild-loader',
+            options: {
+              loader: 'tsx',
+              target: 'es2019',
+            },
+          },
+        },
+        {
+          test: /\.scss$/,
```

Comment:
why are sass and css loaders only added now?

Reply:
previously v8 needed to build everything prior to run. also it was misconfigured to use v9 path aliases etc. more info is in PR description